### ISSUE TITLE
misleading description

### ISSUE THREAD

**Issue by playgithub** [Microsoft/CNTK#1633]
CNTK 101: Logistic Regression and ML Primer
[https://github.com/Microsoft/CNTK/blob/v2.0.beta15.0/Tutorials/CNTK_101_LogisticRegression.ipynb](url)

> Now that the network is setup, we would like to learn the parameters **w** and **b** for our simple linear layer. To do so we convert, the computed evidence (**z**) into a set of predicted probabilities (**p**) using a softmax function.

It means that the function of activation function is to learn the parameters **w** and **b**, it is not true. Actually the activation function is part of the neural network.

**Comment by DanielMerget** [comment]
I guess the sentence refers to defining a loss function without which the rest of the network could not be trained, not to the training process. I agree that it may be misleading, especially for novices. If you have a better and less misleading description, maybe you want to suggest one and/or create a pull request?

**Comment by playgithub** [comment]
In fact, i'm determining which ML framework to use, now between CNTK and tensorflow, I think the quality of the developers are in consideration, somehow very important.

**Comment by DanielMerget** [comment]
The decision between TF and CNTK is of course up to you. Nevertheless, you may want to reconsider your general attitude towards open source projects. If you see something that needs improvement and you already know the (in this case very simple) solution, you should simply fix it and not blame the devs or anyone else for their mistakes. That is not helping anyone. After all, fixing the sentence yourself would take approximately the same amount of time as opening this issue.

**Comment by playgithub** [comment]
I always think that Microsoft's engineering should be better than google's.
Hope Ms can keep its projects excellent.
